docs/releases/3.1.txt

```diff
@@ -96,6 +96,27 @@ and generate and apply a database migration. For now, the old fields and
 transforms are left as a reference to the new ones and are :ref:`deprecated as
 of this release <deprecated-jsonfield>`.
 
+.. _default-hashing-algorithm-usage:
+
+``DEFAULT_HASHING_ALGORITHM`` settings
+--------------------------------------
+
+The new :setting:`DEFAULT_HASHING_ALGORITHM` transitional setting allows
+defining the default hashing algorithm to use for encoding cookies, password
```

Comment:
At the risk of being a pedant, I keep tripping over the word 'defining' here. Here are a couple of suggestions:
'... defining of the...'
'... transitional setting specifies the...'

Reply:
OK, I changed to "specifying".